Handler/Sites.hs

```diff
@@ -22,7 +23,7 @@ getSitesR = do
 
 postSitesR :: Handler Html
 postSitesR = do
-    userId <- requireAuthId
+    Entity userId user <- requireAuth
```

Comment:
If you submit the form from two different sessions at once, you'll create two sites and the button will never be disabled.